src/rofl/common/crofsock.h

```diff
@@ -307,7 +317,7 @@ class crofsock : public cthread_env {
   /**
    *
    */
-  void send_message(rofl::openflow::cofmsg *msg);
+  enum msg_result_t send_message(rofl::openflow::cofmsg *msg, bool enforce_queueing = false, bool keep_message = false);
```

Comment:
`enum` keyword should be removed to be in line with other function returning `msg_result_t`